app/styles/_integrations.less

```diff
@@ -0,0 +1,43 @@
+.service-integration {
+  width: 33%;
+  .image-icon {
+    font-size: 45px;
+    line-height: 1.25;
+    margin: 0 15px;
+    float: left;
+    img {
+      width: 30px;
+    }
+    span {
+      opacity: .38;
+    }
+  };
+
+  .description {
+    float: left;
+  }
+  .id {
+    color: #666666;
```

Comment:
There are color variables as well. You might check if any can be resued -> https://github.com/openshift/origin-web-console/blob/9b8c14669bbcb8723afa44767bcd2930a3b697ad/app/styles/_variables.less

Reply:
I copied the hexadecimal values from elsewhere in the HTML, I'm not seeing any variables that match the values in that file though.